x-pack/plugins/reporting/server/export_types/common/get_conditional_headers.test.ts

```diff
@@ -134,25 +129,12 @@ test(`uses basePath from server if job doesn't have a basePath when creating sav
 });
 
 describe('config formatting', () => {
-  test(`lowercases server.host`, async () => {
-    const mockConfigGet = sinon.stub().withArgs('server', 'host').returns('COOL-HOSTNAME');
```

Comment:
This test runs the same code as the one below it. It was only passing because of the an odd way the stub function worked.